src/gretel_client/client.py

```diff
@@ -345,15 +391,22 @@ def __iter_records(
         start_key = "oldest" if forward else "newest"
         records_seen = 0
         last = position
+
+        # setup base params for streams request
+        headers, params = self._headers_params_from_kwargs(**kwargs)
+        params.update({"with_meta": "yes", "flatten": "yes"})
+        if entity_stream:
```

Comment:
nit: can we do a `if entity_stream is not None` here, in the off chance an empty string is provided, etc.

Reply:
Updated.